tensorflow_datasets/structured/heart_disease.py

```diff
@@ -0,0 +1,85 @@
+"""heart_disease dataset."""
```

Comment:
Please don't remove the License Text at the start of the document.

Reply:
addressed